db/migrate/20161021025012_add_prefrences_to_groups.rb

```diff
@@ -0,0 +1,7 @@
+class AddPrefrencesToGroups < ActiveRecord::Migration
+  def change
+    add_column :groups, :first_choice, :string
+    add_column :groups, :second_choice, :string
+    add_column :groups, :third_choice, :string
+  end
```

Comment:
What are the benefits of that? Here's what I think are the cons:

1) It's more intuitive to say group[:first_choice] than to say group[:preferences][0]
2) You have to worry about the size of the array. Is it always either 0 or 3 (What if someone writes buggy code that adjusts the size of the array?)? Do you handle separately the case where it's size 0 versus size 3?